src/pages/index.tsx

```diff
@@ -33,7 +33,7 @@ const dummyItems = [
       'https://st3.depositphotos.com/15648834/17930/v/600/depositphotos_179308454-stock-illustration-unknown-person-silhouette-glasses-profile.jpg',
   },
   {
-    id: '123',
+    id: '4212',
```

Comment:
if these ID changes are intended to remove duplicate key warnings, they won't fix that issue because we render each dummy item multiple times to fill the home page
but it's not a problem since real data'll have unique IDs  🙂